drop-storage/migrations/20230406110818_initial_migration.sql

```diff
@@ -113,6 +113,7 @@ CREATE TABLE IF NOT EXISTS incoming_path_pending_states (
 CREATE TABLE IF NOT EXISTS incoming_path_started_states (
   id INTEGER PRIMARY KEY AUTOINCREMENT, 
   path_id INTEGER NOT NULL,
+  base_dir TEXT NOT NULL,
```

Comment:
Not sure if this is the optimal naming, suggestions are welcome

Reply:
I would go for `dest_dir` but it's a soft suggestion as it's fine already